### DISCUSSION TITLE
Janus server configuration

### DISCUSSION THREAD

**Discussion by Unique0o** [GetStream/webrtc-android#66] [Q&A]
Hey all, I am curious about finding if we can configure Janus server while using WebRtc using this SDK. I need to develop an Native Android application while our website and backend has already been built utilizing Janus server so to maintain the consistency and for synchronization it is compulsory to configure our WebRtc with Janus server. I would like to know if that works or not. Any help would be highly appreciated. Thank you!

**Comment by 0XiSi** [comment]
Hello my friend, I hope you've found your answer by now, I'm searching to do the same thing and I can't find any clear guide and I have no idea which SDK should I use for it
